### PR TITLE
Update supported client matrix at install guide

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds Oracle OS to supported clients matrix in install guide (already exists in Client Cfg), updates CentOS, updates RHEL8.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12494

### CODE REVIEW

**Comment by Loquacity** [comment]
image

**Comment by Loquacity** [comment]
image

**Comment by Loquacity** [comment]
image

**Review by keichwa — APPROVED** [review]

**Review by juliogonzalez — APPROVED** [review]
Approved, but I saw yet another bug.

CentOS8 [does not have x86 (32 bits)](https://www.centos.org/download/), and I suspect it's the same case for RHEL8 and Oracle8

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
In fact, revoking my approval. The architectures for Oracle are wrong, you only mention x86 in all cases.

If I am not wrong we support:

x86 and x86_64 for RES6, CentOS6, Oracle6
x86_64 for RES7/8, CentOS7/8 and Oracle7/8

But please check SCC to be 100% sure

File: modules/installation/pages/client-requirements.adoc [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 Supported operating systems for traditional and Salt clients are listed in this table.
 
-In this table, icon:check[role="green"] indicates that clients running the operating system are supported by {suse}, and icon:times[role="danger"] indicates that it is not supported.
-Fields marked as icon:question[role="gray"] are under consideration, and may or may not be supported at a later date.
+In this table, {check} indicates that clients running the operating system are supported by {suse}, and {cross} indicates that it is not supported.
+Fields marked as {question} are under consideration, and may or may not be supported at a later date.
 
 .Supported Versions and SP Levels
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:9]: I would change that "note" below a bit. This apply only for SUSE (maintained) Operating Systems.
But this is not true for other Operating Systems SUSE Manager support.
Native RHEL, CentOS, Oracle Linux, Ubuntu: here we only support the latest available version under general support.

File: modules/installation/pages/client-requirements.adoc [hunk 2]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {check}
+| {check}
+
+| {rhel} 6
+| {x86}, {x86}_64
+| {check}
+| {check}
+
+| CentOS 8
+| {x86}, {x86}_64
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:77]: No 32-Bit - At least not for official SUMA support (unknown about Uyuni)

File: modules/installation/pages/client-requirements.adoc [hunk 3]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {cross}
+| {check}
+
+| CentOS 7
+| {x86}, {x86}_64
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:82]: No 32-Bit - At least not for official SUMA support (unknown about Uyuni)

File: modules/installation/pages/client-requirements.adoc [hunk 4]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {check}
+| {check}
+
+| CentOS 6
+| {x86}, {x86}_64
+| {check}
+| {check}
+
+| {oracle}{nbsp}8
+| {x86}
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:92]:
```suggestion
| {x86}_64
```

File: modules/installation/pages/client-requirements.adoc [hunk 5]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {cross}
+| {check}
+
+| {oracle}{nbsp}7
+| {x86}
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:97]:
```suggestion
| {x86}_64
```

File: modules/installation/pages/client-requirements.adoc [hunk 6]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {check}
+| {check}
+
+| {oracle}{nbsp}6
+| {x86}
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:102]:
```suggestion
| {x86}, {x86}_64
```

File: modules/installation/pages/client-requirements.adoc [hunk 7]
```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
...
+| {check}
+| {check}
+
+| {sle} 12
+| {x86}_64, {ppc}, {ibmz}, ARM
+| {check}
+| {check}
+
+| {sle} 11
+| {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc:37]: What is {ipf}? ia64? This is out of regular support and we do not offer LTSS for it. So we also build no client tools for it. So this is not supported.

**Review by mcalmer — CHANGES_REQUESTED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/installation/pages/client-requirements.adoc (hunk 7 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc:37]: ipf: Itanium

**Comment by Loquacity** [comment]
image

**Comment by Loquacity** [comment]
@jcayouette are you able to explain why the bottom line is missing off my table?

File: modules/installation/pages/client-requirements.adoc (hunk 7 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: Itanium is EOL. It does not have a LTSS Channel.

**Review by mcalmer — APPROVED** [review]

File: modules/installation/pages/client-requirements.adoc [hunk 8]
```diff
@@ -3,38 +3,128 @@
...
+| {check}
+| {check}
+
+| {sles}-ES 7
+| {x86_64}
+| {check}
+| {check}
+
+| {sles}-ES 6
+| {x86_64}
```
Comment (juliogonzalez) [inline modules/installation/pages/client-requirements.adoc:54]: All I can find about SLES-ES supported architectures is now at https://www.suse.com/media/data-sheet/sles_with_expanded_support_data_sheet.pdf, can't find anymore where it said that both x86 and x86_64 were supported but only for 6, while 7 and 8 are x86_64 only.

@mcalmer, can you confirm?

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
I think SLES ES6 is still wrong, but I'd like @mcalmer to confirm

File: modules/installation/pages/client-requirements.adoc (hunk 8 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc:54]: Version 6 should have both x86 and x86_64. These are the channels which exists. There are no i386 channels for 7 and 8.

**Review by juliogonzalez — APPROVED** [review]
I guess it's fine, but without a render.. :-D

**Comment by Loquacity** [comment]
> I guess it's fine, but without a render.. :-D

I didn't think you'd want another render for that one small change, but since you asked, here you go ...

image

**Comment by juliogonzalez** [comment]
LGTM. Just a suggestion for the future: group systems by type.

I'd move openSUSE Leap 15 and SLES for SAP right below SLE.

That way SLE, SLES for SAP and Leap will be together, then SLES ES (RES), RES, CentOS and Oracle together, and finally Ubuntu (and in the future Debian) together.

But no need to change it for this PR, and it's only a suggestion.

**Pull request merged**